### DISCUSSION TITLE
Guidance for Getting Started and Community Channel Details

### DISCUSSION THREAD

**Discussion by MAYANK12SHARMA** [JdeRobot/jderobot.github.io#287] [Q&A]
Hii Mentors,

I’m eager to contribute to the organization and would appreciate your guidance. Could you please let me know:

 1. Where should I start to get familiar with the project and its contribution guidelines?
2. Which community channel(s) should I join for updates, discussions, and support?

Thank you!

Best regards,
Mayank Sharma

**Comment by jmplaza** [comment]
Hi @MAYANK12SHARMA ,

the best way to start contributing is to use our open source software and to submit PRs solving open issues.

1.- JdeRobot supports several software projects. Maybe the most relevant one is [RoboticsAcademy](https://jderobot.github.io/RoboticsAcademy/), take a look at [its repository](https://github.com/JdeRobot/RoboticsAcademy) and its documentation (in the docs directory).

2.- The best channel for discussion is the "Discussions" section on that repo, and creating issues there if needed, commenting them and submitting the corresponding PRs. For long term contributors we have a Slack channel for asynchronous and fast communication. 

Cheers,

**Reply by MAYANK12SHARMA** [reply]
Thanks sir for replying, could you please provide me slack channel link so that I can ask questions in future, if I face any problem 🙏.

**Reply by jmplaza** [reply]
I'm sorry @MAYANK12SHARMA, no. The slack channel is only for consolidated and long term contributors. Please use the Discussions, Issues and PullRequest if you need support, and read the documentation. You will receive support there.

Cheers,
